example/README.md

```diff
@@ -1,28 +1,45 @@
 # Amplify Flutter Example
 
-Sample application for getting started with the Amplify Flutter Library
-
-## Getting Started
+Sample flutter app for getting started with the Amplify Flutter Library.  This example uses the Auth, Analytics, and Storage components of the Flutter library. 
 
 Please check out our docs here:
 https://docs.amplify.aws/start/q/integration/flutter
 
-!!! IMPORTANT !!!
+# TL;DR 
+
+**The app will not compile.  You must configure 'amplifyconfiguration.dart' using Amplify CLI.**
+
+**Operations such as logging in or uploading images takes time.  Don't click on the app until they complete.**
+
+
+## Getting Started
+
+**IMPORTANT** 
 
-The app will not compile until you have used the Amplify CLI to configure the
+**The app will not compile until you have used the Amplify CLI to configure** the
 AWS resources necessary for running this app.
 This is because it does not contain an `amplifyconfiguration.dart` file necessary for connecting with AWS services.
 
 You will need to use Amplify CLi to configure Analytics, Auth, and Storage. Make sure you are using the flutter-preview version of the CLI:
 
   npm install -g @aws-amplify/cli@flutter-preview
 
-Afterwards, run `amplify configure` for this project, then `add analytics`, `auth`, and `storage` before pushing.
-If you haven't used the Amplify CLI before, please follow the link below on using Amplify CLI to configure a flutter project:
+If you haven't used Amplify CLI before, please follow the link below on setting up Amplify CLI to configure a flutter project:
 
 Please see the docs here:
-https://docs.amplify.aws/lib/project-setup/create-application/q/platform/flutter#n3-provision-the-backend-with-amplify-cli
+https://docs.amplify.aws/start/getting-started/add-api/q/integration/flutter#setup-aws-cloud-resources-with-amplify-cli
+
+Afterwards, run `amplify configure` for this project, then call `add analytics`, `add auth`, and `add storage` before pushing.
 
 Running these steps will generate the `amplifyconfiguration.dart` file within the `lib` folder. 
 
+## Important Notes 
+
+This is a very basic app that interacts with AWS resources.  We did not implement UI showing that the app is "loading" or "uploading" something from AWS.  Some operations like logging in or uploading an image can take some time.  If you do an action and the app doesn't immediately respond, please refrain from clicking away or trying to click a button again - the operation is in progress and interacting with the UI can interfere with that.    
```

Comment:
I would shorten this to "This is a very basic app that interacts with AWS resources.  We did not implement UI showing that the app is "loading" or "uploading" something from AWS.  Some operations like logging in or uploading an image can take some time. "

Reply:
done